lib/Varien/Image/Adapter/Imagick.php

```diff
@@ -0,0 +1,250 @@
+<?php
+declare(strict_types=1);
```

Comment:
should we have the strict_types? there's only in one single file in the whole repo, isn't it better just not to use it?=